libs/block-editor/src/lib/extensions/services/suggestions/suggestions.service.ts

```diff
@@ -25,10 +26,15 @@ export class SuggestionsService {
             .pipe(pluck('entity'));
     }
 
-    getContentlets(contentType = '', filter = ''): Observable<DotCMSContentlet[]> {
+    // TODO: change the contentType param to use an Enum
```

Comment:
the content type can't be an enum because is basically what ever the use wants to name the contents.